draft/draft-ietf-bfd-rfc9127.xml

```diff
@@ -3250,5 +3250,19 @@ module example-bfd-echo {
         </address>
       </author>
     </section>
+    <section anchor="updates-since-9127" numbered="false" removeInRFC="false" toc="include" pn="section-appendix.d">
+      <name slugifiedName="updates">Updates since RFC 9127</name>
+      <t>This version of the draft updates the 'ietf-types-module'
```

Comment:
Should instead be: This version of the draft updates the 'ietf-bfd-types' module?